plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -74,7 +105,46 @@ public override IEnumerable<FileModel> Prebuild(ImmutableList<FileModel> models,
                      SourceFiles = new FileAndType[] { item.Value.Item1 }.ToImmutableList(),
                  }).ToImmutableList();
 
-            return collection;
+            return modelsDict.Values.ToList();
```

Comment:
This method's return type is `IEnumerable`, while Values(of type `ValueCollection`) already implements this, so not need to `ToList()` here.